src/Polly.Pcl/project.json

```diff
@@ -1,12 +1,7 @@
 ﻿{
-    "dependencies": {
-        "NuSpec.ReferenceGenerator": {
-            "version": "1.4.0",
-            "type": "build"
-        }
-    },
-    "frameworks": {
-        ".NETPortable,Version=v4.5,Profile=Profile259": { }
-    },
-    "supports": {}
-}
+  "dependencies": {},
+  "frameworks": {
+    ".NETPortable,Version=v4.5,Profile=Profile259": {}
```

Comment:
:+1: Thanks @twsouthwick .  Will definitely revert to you if we have any qs when the VS2017 tooling comes out.  Great to have support direct from the MS teams.